free-programming-books-tr.md

```diff
@@ -139,7 +139,7 @@
 
 * [AB2014 Ruby Programlama Dili](https://github.com/leylaKapi/AB2014-Ruby-Programlama-Dili/blob/master/Ruby_AB2014.md)
 * [Ruby](https://www.ruby-lang.org/tr/)
-* [Ruby 101](https://www.gitbook.io/book/vigo/ruby-101)
+* [Ruby 101](https://vigo.gitbooks.io/ruby-101/content/)
```

Comment:
Ok, but it means it'll redirect to https://www.gitbook.com/book/vigo/ruby-101/details